src/components/Hoverable/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * It is necessary to create a Hoverable component instead of relying solely on Pressable support for hover state,
+ * because nesting Pressables causes issues. https://github.com/necolas/react-native-web/issues/1875
+ */
+import React, {Component} from 'react';
+import {propTypes, defaultProps} from './HoverablePropTypes';
+
+class Hoverable extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isHovered: false,
+        };
+        this.handleMouseEnter = this.handleMouseEnter.bind(this);
+        this.handleMouseLeave = this.handleMouseLeave.bind(this);
+    }
+
+    handleMouseEnter() {
+        if (!this.state.isHovered) {
+            this.props.onHoverIn();
```

Comment:
Not seeing where this is used.

Reply:
True, these aren't used. I _could_ remove them for now, but this implementation was more-or-less copied from the author of `react-native-web`. I think having `onHoverIn` and `onHoverOut` props is simple enough and has enough potential for future use that we should just them for now.